plugins/pulp_rpm/plugins/importers/yum/pulp_solv.py

```diff
@@ -371,16 +371,71 @@ def load(self):
             _LOGGER.info('Loaded target repository %s', self.target_repo.repo_id)
 
     def _units_jobs(self, units):
+        # disfavor dummies first
+        flags = solv.Job.SOLVER_DISFAVOR | solv.Job.SOLVER_SOLVABLE
```

Comment:
I thought `SOLVER_DISFAVOR` would be required to skip the dummies in case next unit being copied would conflict with what a dummy provides. Since we don't track the conflicts, this isn't needed here (and conflicts can be handled with `considered` masking later on too). @ignatenkobrain  would you agree?